history-of-quakers/modernized/07-chapter-5.adoc

```diff
@@ -891,6 +949,9 @@ but also once in the ship Nightingale,
 ignominiously exposed with a quarter can about his neck,
 as appears by the following testimonies.
 
+[.embedded-content-document]
```

Comment:
I didn't label these as [testimony] because there aren't in the Quaker sense of testimony, but the text does call them that so idk if that's right.

Reply:
i think i agree with not adding `testimony`.